crates/aptos/src/common/types.rs

```diff
@@ -1176,7 +1176,7 @@ pub struct MovePackageDir {
     /// Currently, defaults to `1`, unless `--move-2` is selected.
     #[clap(long, value_parser = clap::value_parser!(LanguageVersion),
            alias = "language",
-           default_value_if("move_2", "true", "2.0"),
+           default_value_if("move_2", "true", "2.1"),
```

Comment:
Better to change this to "2" and change third_party/move/move-model/src/metadata.rs to interpret "2" as "2.1" since you're making that the default now.

Reply:
Not in this PR, and the right solution is using programmatically `options.move_2()`, who knows which other flags we need to eventually set.